spaces/domains.json

```
@@ -1,4 +1,5 @@
-{
+{ 
+  "voting.cafeswap.finance":"cafeswapfinance.eth",
```

Just one space is missing and it's all good.
```suggestion
  "voting.cafeswap.finance": "cafeswapfinance.eth",
```